src/number.js

```diff
@@ -19,11 +20,11 @@ define([
 
 	"cldr/event",
 	"cldr/supplemental"
-], function( Globalize, createErrorUnsupportedFeature, validateCldr, validateDefaultLocale,
-	validateParameterPresence, validateParameterRange, validateParameterTypeNumber,
-	validateParameterTypePlainObject, validateParameterTypeString, numberFormat,
-	numberFormatProperties, numberNumberingSystem, numberParse, numberParseProperties,
-	numberPattern, numberSymbol, stringPad ) {
+], function( Globalize, createErrorUnsupportedFeature, runtimeBind, validateCldr,
```

Comment:
Btw. having worked with ES2015 modules (with babel for transpiling) for 3 months, AMD define is pretty ugly in comparison... Looking forward to switching here.